src/test/java/org/ethereum/vm/TestBase.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2017-2018 The Semux Developers
+ *
+ * Distributed under the MIT software license, see the accompanying file
+ * LICENSE or https://opensource.org/licenses/mit-license.php
+ */
+package org.ethereum.vm;
+
+import java.math.BigInteger;
+import java.util.Arrays;
+
+import org.ethereum.vm.client.BlockStore;
+import org.ethereum.vm.client.BlockStoreMockImpl;
+import org.ethereum.vm.client.Repository;
+import org.ethereum.vm.client.RepositoryMockImpl;
+import org.ethereum.vm.program.Program;
+import org.ethereum.vm.program.invoke.ProgramInvokeImpl;
+import org.ethereum.vm.util.ByteArrayUtil;
+import org.junit.After;
+import org.junit.Before;
+
+public class TestBase {
+
+    protected byte[] address = address(1);
```

Comment:
`byte[]` was purposely used to mimic the real data types in the `Transaction` data structure.

Reply:
Hmm, but the first thing you do with it is to wrap it with `DataWord`. 

BTW: I am wondering what is `DataWord` class for? If it's just to serve the `and`, `or`, `xor`, etc… methods, then they could as well be static. I am asking, because here you use `byte[]`, then later (in `@Before`) you wrap it.